### DISCUSSION TITLE
Impressions, issues, potential pull request

### DISCUSSION THREAD

**Discussion by naixx** [seerge/g-helper#1536] [Ideas]
I've installed and it is really great! I'm so happy that uninstalled Asus stuff, my Zephyrus S17 works as quite as it should two years ago when I bought it. Also noticed that when disabling CPU boost laptop doesn't get hot when in sleep mode. That was a nice observation.

1. I notice that on a heavy load, G-Helper shows higher temperatures than it was shown in Armory crate. Is that just visualization or real data? Or is that some settings issue? 
2. I really liked that AC showed CPU frequency, so I've made a modification to the app that replaces the default tray icon to the icon with some useful info. The idea was taken from ThrottleStop which showed temperature in tray. Now G-Helper can show temperature and clock speed in tray, not just "another icon". I'm not creating a PR because a) probably you have another vision for the app b) I'm not a .net dev, so code changes can require some modifications, however I've checked them for memory leaks, should be fine. Right now it looks like this
![image](https://github.com/seerge/g-helper/assets/1759282/8f3c49a9-82a5-4fa1-ac95-55e007132e78)

**Comment by JV-bug** [comment]
Hello, this app is great, clean, and pleasure to use.
But what made me jump back to ACrate is that it can control P and E cores. This is kind of really needed because of Ubisoft, all their titles AC Odyssey, Valhalla, Mirage go bananas with microstutters when it comes to E Cores, tried all i could, but the only way is to disable E Cores, and sadly Zephyrus M16 2023 i9-13900H, RTX 40XX have no such option to disable E Cores in BIOS, only disableing Hyper Threading, which actually doesnt solve the Ubisoft titles issue. Tried Process Lasso and disabling E Cores via TaskManager, those doesnt seem to do it properly, but once done via ACrate all is smooth, could this option please be considered as future development for g-helper?

**Reply by IceStormNG** [reply]
I have a 2023 M16 and it does have the option to disable E-Cores in BIOS.
you can also change the windows scheduler to prefer P-Cores which did resolve most of the stuttering for me.

**Reply by JV-bug** [reply]
Helped. Thank you.
